### ISSUE TITLE
Cluster-level logging architecture

### ISSUE THREAD

**Issue by wolfm89** [schul-cloud/jitsi-deployment#9]
There is no setup of any logging architecture right now. Streamlining of all logs, persistence, and accessibility is important as it gives us possibility to analyze anomalies and failures after their occurrences.

**Comment by mvakert** [comment]
The retention time of the logs is 7 days and those of metrics 1 day by default, as mentioned [here](https://www.elastic.co/guide/en/cloud-enterprise/current/ece-monitoring-ece-set-retention.html). However, one can change this using a [Curator](https://github.com/helm/charts/tree/master/stable/elasticsearch-curator).

**Comment by wolfm89** [comment]
A basic cluster-wide logging architecture has been set up using [ECK](https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-overview.html) and can be found in `base/ops/logging`. We use Fluentd + Elasticsearch + Kibana.
